nncf/openvino/graph/model_utils.py

```diff
@@ -64,24 +64,18 @@ def get_start_nodes_for_activation_path_tracing(nncf_graph: NNCFGraph) -> List[N
 
 def remove_friendly_name_duplicates(model: ov.Model) -> ov.Model:
     """
-    Removes diplicates of node names (friendly_name attribute) in the model.
+    Removes duplicates of node names (friendly_name attribute) in the model.
 
     :param model: ov.Model instance to update.
     :return: Updated ov.Model without duplicated names.
     """
-    rt_info_path = ["nncf", "friendly_names_were_updated"]
-    friendly_names_flag = "True"
-    if model.has_rt_info(rt_info_path) and model.get_rt_info(rt_info_path).value == friendly_names_flag:
-        return model
-
```

Comment:
We should not remove this due to performance issues.
We should fix the original reason for the duplicate's existence.
cc @alexsu52

Reply:
> We should not remove this due to performance issues.
I don't see any performance issues. According to my tests the extra for loops takes about 1 ms per 1000 model nodes. 

> We should fix the original reason for the duplicate's existence.
I agree, but this is a temporary workaround while the investigation is in progress.